### PR TITLE
Add items to Subprojects section

### PR DESCRIPTION
**Pull request by carlossvg**
# Working Group Modification Pull Request

# Add Project

This MR adds some projects to the subprojects section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,62 @@ Its meetings and membership are largely focused on the direction, design, and wo
...
+- realtime_support
+  - Description: Minimal real-time testing utility for measuring jitter and latency
+    - rttest: rttest is a minimal tool for instrumenting and running tests for synchronous real-time systems
+    - tlsf_cpp: C++ stdlib-compatible wrapper around tlsf allocator and ROS2 examples
+  - Repositories
+    - https://github.com/ros2/realtime_support
+- ros2_tracing
+  - Description: Tracing tools for ROS 2
+  - Repositories
+
```
Comment (christophebedard) [inline README.md:51]: removing this empty line will fix the weird vertical spacing: https://github.com/ros-realtime/community/blob/add-subprojects/README.md

**Review by christophebedard — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -11,11 +11,62 @@ Its meetings and membership are largely focused on the direction, design, and wo
...
+- ros2-performance
+  - Description: iRobot ROS2 performance evaluation framework
+  - Repositories
+    - https://github.com/irobot-ros/ros2-performance
+- TwoWaysMeasurement
+  - Description: Tool to test the real-time performance in a ping-pong scenario
+  - https://github.com/y-okumura-isp/TwoWaysMeasurement
+- ros2_timer_latency_measurement
+  - Description: Tool to measure the accuracy of the ROS 2 timer
+  - https://github.com/hsgwa/ros2_timer_latency_measurement
```
Comment (christophebedard) [inline README.md:38]: a bit nitpicky, but the two items above don't have a `- Repositories` line before the repo link (unlike all the other items).

File: README.md (hunk 1 above)
Comment (carlossvg) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (carlossvg) [reply README.md]: Fixed. Thanks for the review!

**Review by christophebedard — APPROVED** [review]
I can't think of anything that's missing. In any case, we can always add more projects later on!

**Comment by christophebedard** [comment]
oh wait what about https://github.com/osrf/osrf_testing_tools_cpp?

**Comment by carlossvg** [comment]
> oh wait what about https://github.com/osrf/osrf_testing_tools_cpp?

Good catch! I think we should add these too:
- https://gitlab.com/ApexAI/apex_containers 
- https://gitlab.com/ApexAI/apex_test_tools

**Comment by christophebedard** [comment]
looks good!

**Comment by razr** [comment]
there are also
https://github.com/ros2/buildfarm_perf_tests
https://github.com/ros2/performance_test

**Performance evaluation** - probably **Performance measurements** is a better name?

**Comment by carlossvg** [comment]
> there are also
> https://github.com/ros2/buildfarm_perf_tests
> https://github.com/ros2/performance_test
> 
> **Performance evaluation** - probably **Performance measurements** is a better name?

Updated

**Comment by carlossvg** [comment]
I added https://github.com/ros-controls/realtime_tools/tree/foxy-devel/include/realtime_tools

**Pull request merged**